woocommerce/woocommerce-android
# [Payment Method Improvements] Gray out button until payment amount is greater or equal to amount due

Closes: #11604


This PR is branched off of https://github.com/woocommerce/woocommerce-android/pull/11594

### Description



**Gray out "Mark order as complete" button:**
   - The "Mark order as complete" button is disabled if the amount of cash received is less than the amount due. This prevents users from completing the order without receiving sufficient payment.

**Display correct currency symbol in change due field:**
   - The change due field now correctly displays the currency symbol, aligning with the currency used for the transaction.


### Testing instructions


0. Enable `OTHER_PAYMENT_METHODS`

1. **Gray out button test:**
   - Navigate to an order that needs to be paid with cash.
   - Enter a cash received amount that is less than the amount due.
   - Verify that the "Mark order as complete" button is grayed out and disabled.
   - Enter a cash received amount that is equal to or greater than the amount due.
   - Verify that the "Mark order as complete" button is enabled.

2. **Currency symbol in change due field test:**
   - Verify that the change due field displays the correct currency symbol as defined by the app's currency settings.



---
